nss-day-cohort-46/holiday-road-accidental-engine
# Nah preview testing

# Description
I targeted some specific containers on index.html for the details data and the dropdown data.
It should help make it easier to read on the DOM and improve some fucntionality.
We still don't have attractionDetails rendering to the DOM correctly but everything else looks good.


# Testing Instructions
git fetch --all
git checkout nah-PreviewTesting
git serve

you can click an item on any dropdown and see it render to the preview section on the DOM.
if you choose a new item from the dropdown, it replaces the current one.
You can also see the details button below the selected item and once clicked, it populates to the DOM below the name of the selected item.

# Checklist:

- [X] My changes generate no new warnings or errors // attractionDetails is not working still....
